### ISSUE TITLE
Error compiling last release using hex

### ISSUE THREAD

**Issue by VitorTrin** [silviucpp/erlkaf#55]
When I use hex and I  add a dependency to the latest release like this
```elixir
{:erlkaf, "~> 2.1.0"}
```

fetching the deps results in the following error

```
Error evaluating Rebar config script ./rebar.config.script:22: evaluation failed with reason error:{badmatch,{error,enoent}} and stacktrace [{erl_eval,expr,6,[{file,[101,114,108,95,101,118,97,108,46,101,114,108]},{line,496}]},{erl_eval,exprs,6,[{file,[101,114,108,95,101,118,97,108,46,101,114,108]},{line,136}]},{erl_eval,expr_list,7,[{file,[101,114,108,95,101,118,97,108,46,101,114,108]},{line,961}]},{erl_eval,expr,6,[{file,[101,114,108,95,101,118,97,108,46,101,114,108]},{line,472}]},{file,eval_stream2,6,[{file,[102,105,108,101,46,101,114,108]},{line,1504}]},{file,script,2,[{file,[102,105,108,101,46,101,114,108]},{line,1142}]},{'Elixir.File','cd!',2,[{file,[108,105,98,47,102,105,108,101,46,101,120]},{line,1607}]},{'Elixir.Mix.Rebar',eval_script,2,[{file,[108,105,98,47,109,105,120,47,114,101,98,97,114,46,101,120]},{line,191}]}]
Any dependencies defined in the script won't be available unless you add them to your Mix project
```
This does not happen if we add the dependency directly from the github like this: 
```elixir
{:erlkaf, github: "silviucpp/erlkaf"}
```

even though no changes happened to rebar config in the master since the last release

**Comment by silviucpp** [comment]
Hello Victor. 

Can you please check if when you fetch using `{:erlkaf, "~> 2.1.0"}` you have `rebar.config.script` in the packet ?

This file shouldn't be there... I have no explanation how arrived in the packet..

**Comment by silviucpp** [comment]
Hmm.. I have no clue to what rebar.config.script makes reference.. because mine has less than 22 lines.. and also I don't see it included into the packet.. 

Have you tried to update to last rebar3 ?

**Comment by VitorTrin** [comment]
`rebar3 --version` gives me `rebar 3.15.2 on Erlang/OTP 25 Erts 13.1.3`
and surprisingly, no the file does not appear to be in the packet. This line 22 thing smells like a macro at work that expands the file

**Comment by VitorTrin** [comment]
It's probably https://github.com/fogfish/esq/issues/20

**Comment by silviucpp** [comment]
You are totally right ! But I don't see how can I fix this... I have no permissions on that library to update the hex package..

Silviu

**Comment by silviucpp** [comment]
@VitorTrin can you try `{:erlkaf, "~> 2.1.1"}` . I just published it. 

Silviu

**Comment by silviucpp** [comment]
I published `{:erlkaf, "~> 2.1.2"}` as another problem with esq appeared.

**Comment by silviucpp** [comment]
Already received confirmation from another user that problem is solved.
